velox/dwio/dwrf/test/TestReader.cpp

```diff
@@ -505,6 +505,47 @@ TEST(TestReader, testStatsCallbackFiredWithFiltering) {
   EXPECT_EQ(selectedKeyStreamsAggregate, 4);
 }
 
+TEST(TestReader, testEstimatedSize) {
+  const std::string fmSmall(getExampleFilePath("fm_small.orc"));
+
```

Comment:
NYC: could you help rename this file to ReaderTest.cpp to follow the other test file naming? Thanks!